public/wp-content/themes/newsup-child/single-genre.php

```diff
@@ -7,169 +7,229 @@
 <!-- =========================
      Page Content Section
 ============================== -->
+<!--<p>single-anime.php -> in child dir</p>-->
```

Comment:
Remember to remove all your test code

Reply:
I will be going thru a full documentation overhaul after these round of branches are done.

Reply:
done